### DISCUSSION TITLE
How to make lsp context text to use the same color as the symbol in front of it?

### DISCUSSION THREAD

**Discussion by hopezh** [nvim-lualine/lualine.nvim#1047] [Q&A] [answered]
May I ask how to make the color of the context text, as generated from lsp, to follow that of the symbol in front of it? (Barbecue seems to have an option to toggle this.) 

Thanks. 

![image](https://github.com/nvim-lualine/lualine.nvim/assets/23435810/4d9acd4b-ef9f-42b0-82da-437992ad7349)

**Comment by fitrh** [comment, marked as answer]
I don't think lualine have that component built-in, so it depends on what you are used for creating that component, maybe you can share that component so we can help you

**Reply by hopezh** [reply]
Thanks. I don't have a particular application scenario in which such a feature is very useful, or important.
